mirth/docker/docker-compose-mirth.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  mc:
+    image: nextgenhealthcare/connect:4.5.2
+    platform: linux/amd64
+    restart: always
```

Comment:
I think this is what we want? (So that the containers always restart on server restarts/power outages/etc?)

Reply:
Yes, that seems correct